WordPress/Classes/Utility/Blogging Reminders/BloggingRemindersScheduler.swift

```diff
@@ -141,6 +141,59 @@ class BloggingRemindersScheduler {
             .appendingPathComponent(defaultDataFileName)
     }
 
+    private static func sharedDataFileURL() -> URL? {
+        let sharedDirectory = FileManager.default.containerURL(forSecurityApplicationGroupIdentifier: "group.org.wordpress")
+        return sharedDirectory?.appendingPathComponent(defaultDataFileName)
+    }
+
+    static func handleRemindersMigration() {
+        guard FeatureFlag.sharedLogin.enabled else {
+            return
+        }
+
+        if AppConfiguration.isWordPress {
+            copyStoreToSharedFile()
+        } else if AppConfiguration.isJetpack {
+            copyStoreToLocalFile()
+        }
+    }
+
+    private static func copyStoreToSharedFile() {
+        guard !UserDefaults.standard.bool(forKey: "reminders-copied"),
```

Comment:
I used `standard` here because it's specific to the app you're using. I think either way it wouldn't actually matter if I used the shared defaults vs the local one here. Since that's the case, I'll make that change along with using `UserPersistentRepositoryUtility`.

Reply:
Hey, I meant to add a variable as you've done but use `standard` in it instead of factory so that we know it will always fetch the local/non-shared one. If either way is fine, we can go ahead with the current state too. I don't know the requirements here so I wouldn't want to affect the logic there

Reply:
The keys are already separate so it being shared should be fine. I figured I'd just make it shared to be consistent.